.github/workflows/ci.yml

```diff
@@ -193,6 +195,48 @@ jobs:
           check-repo: false
           ignore-unpublished-changes: true
 
+  publish-codecov:
+    name: Publish code coverage report on GitHub pages branch
+    runs-on: buildjet-4vcpu-ubuntu-2204
+    needs:
+      - cargo-verifications
```

Comment:
In the vm pr we have `needs: -cargo`. What's the difference?

Reply:
This workflow doesn't have `cargo` job. The closes equivalent is `cargo-verifications`.

Reply:
Okay. That makes sense. Do we have any issues for unifying them?